src/widget/OktaSignIn.js

```diff
@@ -12,7 +12,7 @@ var OktaSignIn = (function () {
      * @param callback - callback function invoked with 'true'/'false' as the argument.
      */
     function checkSession(callback) {
-      authClient.existingSession().then(callback);
+      authClient.session.exists().then(callback);
```

Comment:
nit: the name of this method is a little bit confusing ```session.exists()``` sounds like a method to know if the session exists or not, that will return a boolean. Maybe a better one will be ```session.current()``` or something similar?